src/System.AppContext/ref/System.AppContext.csproj

```diff
@@ -4,8 +4,8 @@
   <PropertyGroup>
     <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>netstandard1.5</PackageTargetFramework>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
+    <PackageTargetFramework>netstandard1.6</PackageTargetFramework>
```

Comment:
We only added 2 APIs, and those are going to be part of 4.6.3. I think we only need 1.6 here.